protocols/virtualdefund/virtualdefund.go

```diff
@@ -304,6 +301,20 @@ func (o *Objective) Crank(secretKey *[]byte) (protocols.Objective, protocols.Sid
 		return &updated, sideEffects, WaitingForNothing, protocols.ErrNotApproved
 	}
 
+	// We only send out a voucher if we're Alice and we haven't already sent one
+	if !updated.VoucherSent && updated.MyRole == payments.PAYER_INDEX {
```

Comment:
I think this means that only Alice can start closing a channel 😦 . We should allow Bob/Irene to send out vouchers so they can communicate with others they want to close.